app/models/katello/system.rb

```diff
@@ -63,6 +63,11 @@ class System < Katello::Model
 
     belongs_to :content_view, :inverse_of => :systems
 
+    has_one :capsule,
+            :class_name => "::SmartProxy",
+            :inverse_of => :system,
```

Comment:
I can update the migration to be content_host instead. My brain was just naming things based off the model.